bin/cheritest/cheritest.c

```diff
@@ -1000,7 +1000,9 @@ static const struct cheri_test cheri_tests[] = {
 
 	{ .ct_name = "cheritest_vm_cap_share_sigaction",
 	  .ct_desc = "Demonstrate capability passing via shared sigaction table",
-	  .ct_func = cheritest_vm_cap_share_sigaction, },
+	  .ct_func = cheritest_vm_cap_share_sigaction,
```

Comment:
Maybe drop the original and revert from the branch before pushing?  Is there a good way to mark it as XFAIL for the non-mt cases only?

Reply:
The `-mt` binaries define `CHERITHREAD_TESTS` if that's what you mean?

Reply:
Yeah, though I'd want to understand why threading makes a difference.  It really shouldn't.  Probably fine to use the current approach of always XFAIL since that is what we expect and we aren't (yet) running the -mt tests for PRs (though eventually we should?)